docfx_yaml/extension.py

```diff
@@ -1260,6 +1261,117 @@ def find_markdown_pages(app, outdir):
             })
 
 
+# Finds and replaces occurrences which should be a cross reference in the given
+# content, except for the current name.
+def convert_cross_references(content, current_name, list_of_entry_names):
```

Comment:
Total nit: I think just `entry_names` is enough? Sounds like it would be a list.

Reply:
If you feel it's unclear you could alternatively add type hints here. :)

Reply:
Adding type hints sounds like a good idea! I'll keep this in mind when refactoring the code as well.